packages/core/src/App/Containers/Routes/routes.jsx

```diff
@@ -17,10 +17,21 @@ const Error = Loadable({
 });
 
 class Routes extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            is_bot: false,
+        };
+    }
+    el_landscape_blocker = document.getElementById('landscape_blocker');
     unlisten_to_change = null;
     initial_route = null;
 
     componentDidMount() {
+        this.isBot(window.location);
+        if (this.state.is_bot) {
+            this.el_landscape_blocker.classList.add('landscape-blocker--keyboard-visible');
```

Comment:
Could we refactor `landscape-blocker--keyboard-visible` across the app to be `landscape-blocker--disabled`? In this case we're no longer hiding the landscape blocker due to a keyboard being shown, however they do yield the same result (hiding the landscape blocker).

Reply:
Aaron I've checked it and I think if I use isBot() function from share, we have more latency to remove blocker when we back from a blocked page to dbot

Reply:
> Could we refactor `landscape-blocker--keyboard-visible` across the app to be `landscape-blocker--disabled`? In this case we're no longer hiding the landscape blocker due to a keyboard being shown, however they do yield the same result (hiding the landscape blocker).

Done